src/main.js

```diff
@@ -227,6 +228,22 @@ function showPosterGrid() {
   }
 }
 
+function deletePoster(event) {
+  console.log('mac and cheese');
+    if (event.target(savedPosters)) {
+      for (var i = 0; i < savedPosters.length; i++) {
+        
+        // Uncaught TypeError: event.target is not a function
+        // at HTMLElement.deletePoster (main.js:234)
+
+        if (event.target === savedPosters[i]) {
+          savedPosters.splice(i,1)
+  }
+  }
+}
+}
```

Comment:
I was thinking about it last night, next time if we get to a point where we're stuck but need to push for the end of the night, lets comment the whole section out.